ocaml/tests/suite_alcotest.ml

```diff
@@ -71,4 +71,5 @@ let () =
     @ Test_session.tests
     @ Test_xapi_cmd_result.tests
     @ Test_extauth_plugin_ADwinbind.tests
+    @ Test_pkg_mgr.tests
```

Comment:
Can Test_pkg_mgr be its own test binary instead of being part of the suite? It allows the tests to be run alone easily, and in parallel with other tests.